### ISSUE TITLE
play-in tournament statistics unreported

### ISSUE THREAD

**Issue by engineerchange** [abresler/nbastatR#102]
A data pull from today:
```
> season = nbastatR::game_logs(seasons = 2022,season_types = c("Regular Season","Playoffs"))
> season %>% arrange(desc(dateGame)) %>% dplyr::filter(nameTeam=="San Antonio Spurs")
# A tibble: 910 × 58
   yearSeason slugSeason slugLeague typeSeason     dateGame    idGame numberGameTeamS… nameTeam idTeam isB2B isB2BFirst
        <dbl> <chr>      <chr>      <chr>          <date>       <dbl>            <int> <chr>     <dbl> <lgl> <lgl>     
 1       2022 2021-22    NBA        Regular Season 2022-04-10  2.21e7               82 San Ant… 1.61e9 TRUE  FALSE     
 2       2022 2021-22    NBA        Regular Season 2022-04-10  2.21e7               82 San Ant… 1.61e9 TRUE  FALSE     
 3       2022 2021-22    NBA        Regular Season 2022-04-10  2.21e7               82 San Ant… 1.61e9 TRUE  FALSE 
 ```
 
The Spurs and Pelicans had a game on April 13th for the new play-in tournament, but it doesn't appear that data is output in either the "Regular Season" or "Playoffs" season type.

**Comment by patstout** [comment]
Last year setting the season_type to "PlayIn" worked for me:
game_logs(seasons = 2021, season_types = "PlayIn").
Last year I pointed this out in issue #78

**Comment by engineerchange** [comment]
Thanks! Worked for me.

```
season = nbastatR::game_logs(seasons = 2022,season_types = c("Regular Season","Playoffs","PlayIn"))
> season %>% arrange(desc(dateGame)) %>% dplyr::filter(nameTeam=="San Antonio Spurs")
# A tibble: 920 × 58
   yearSeason slugSeason slugLeague typeSeason dateGame     idGame numberGameTeamSeas… nameTeam idTeam isB2B isB2BFirst
        <dbl> <chr>      <chr>      <chr>      <date>        <dbl>               <int> <chr>     <dbl> <lgl> <lgl>     
 1       2022 2021-22    NBA        PlayIn     2022-04-13 52100131                   1 San Ant… 1.61e9 FALSE FALSE     
 2       2022 2021-22    NBA        PlayIn     2022-04-13 52100131                   1 San Ant… 1.61e9 FALSE FALSE     
 3       2022 2021-22    NBA        PlayIn     2022-04-13 52100131                   1 San Ant… 1.61e9 FALSE FALSE  
```

**Comment by engineerchange** [comment]
Closed as duplicate of #78.
